tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -4,24 +4,53 @@
 #include "Framework/TestingFramework.hpp"
 
 #include <cstddef>
+#include <limits>
 #include <memory>
+#include <tuple>
+#include <unordered_map>
 
 #include "DataStructures/Tensor/TypeAliases.hpp"  // IWYU pragma: keep
 #include "Domain/Block.hpp"                       // IWYU pragma: keep
 #include "Domain/BlockNeighbor.hpp"               // IWYU pragma: keep
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.tpp"
+#include "Domain/CoordinateMaps/ProductMapsTimeDep.hpp"
```

Comment:
the time dependent coordinate maps have been moved to `CoordinateMaps/TimeDependent` and into a namespace